subprojects/platform-play/src/integTest/groovy/org/gradle/play/integtest/continuous/PlayContinuousBuildReloadIntegrationTest.groovy

```diff
@@ -74,6 +84,7 @@ class PlayContinuousBuildReloadIntegrationTest extends AbstractPlayReloadIntegra
         then:
         println "CHECKING ERROR PAGE"
         errorPageHasTaskFailure("compilePlayBinaryScala")
+        waitForBuildFinish()
```

Comment:
Does `waitForBuild()` work here? It seems like it does something similar.

Could the output scraping for pending changes be replaced with the `BlockingHttpServer` like that's used in `PlayContinuousBuildReloadWaitingIntegrationTest` and `PlayReloadWaitingIntegrationTest`?

Reply:
Maybe something like this: 
https://github.com/gradle/gradle/commit/e6a84cb27ac86d581a26e0a5a8a4d1fd1f28172d

It passes locally, but I'm not sure if it's more reliable/stable.